### PR TITLE
Export price qualifier info

### PR DESCRIPTION
**Pull request by johanpoirier**
The `<PriceQualifier>` info was parsed from the ONIX file but never used in the `product`.
We set the info if it's present in the ONIX file.

### CODE REVIEW

File: lib/onix/onix21.rb [hunk 1]
```diff
@@ -107,6 +107,7 @@ def -v
     class Price < SubsetDSL
       element "PriceTypeCode", :subset, :klass => "PriceType"
       element "PriceAmount", :float, {:lambda => lambda { |v| (v*100).round }}
+      element "PriceQualifier", :subset, :klass => "PriceQualifier"
```
Comment (johanpoirier) [inline lib/onix/onix21.rb]: OK, I'll remove that part. Thanks.

**Comment by julbouln** [comment]
onix21.rb is about ONIX 2.1, ONIX 3.0 PriceQualifier is in https://github.com/immateriel/im_onix/blob/3b116f93829779ee7bbbdef9337041d132853da2/lib/onix/price.rb#L11
Do you really intend to modify ONIX 2.1 part ?

**Comment by johanpoirier** [comment]
> Do you really intend to modify ONIX 2.1 part ?

The `<PriceQualifier>` is present in the ONIX for Books 2.1 rev.03 from January 2006 specifications.
Why not add this information in the ONIX 2.1 part? If I remove it, it'll break the code (and the tests).

**Comment by julbouln** [comment]
It is fine for me, it was just to be sure since the ONIX fixture attached to this PR seems to be ONIX 3.0 ?

**Comment by johanpoirier** [comment]
> It is fine for me, it was just to be sure since the ONIX fixture attached to this PR seems to be ONIX 3.0 ?

I didn't test it against a ONIX 2.1 file. I just trust the specs. Maybe I shouldn't, who knows what can happen with ONIX files?

**Pull request merged**